gnomad/utils/constraint.py

```diff
@@ -1284,24 +1284,28 @@ def calculate_raw_z_score_sd(
     return hl.agg.filter(filter_expr, sd_expr)
 
 
-def add_gencode_annotations(
+def add_gencode_transcript_annotations(
     ht: hl.Table,
     gencode_ht: hl.Table,
+    annotations: List[str] = ["level", "transcript_type"],
 ) -> hl.Table:
     """
     Add GENCODE annotations to Table based on transcript id.
 
     .. note::
-        Added annotations are:
-        - chromosome
+        Added annotations by default are:
```

Comment:
maybe should add a note here that the added annotations become keys for the group by (so `cds_length` will get calculated over the combination of all added annotations, for example)